### PR TITLE
Fixed typo and updated some docblocks

### PR DESCRIPTION
**Pull request by itowhid06**

### CODE REVIEW

File: classes/class-twentytwenty-customize.php [hunk 1]
```diff
@@ -420,7 +422,9 @@ public static function sanitize_select( $input, $setting ) {
 		/**
 		 * Sanitize boolean for checkbox.
 		 *
-		 * @param bool $checked Wethere or not a blox is checked.
+		 * @param bool $checked Wethere or not a box is checked.
```
Comment (ntwb) [inline classes/class-twentytwenty-customize.php]: Spelling: Whether

File: inc/template-tags.php [hunk 2]
```diff
@@ -26,6 +26,8 @@
  *
  * @param array   $args Arguments for displaying the site logo either as an image or text.
  * @param boolean $echo Echo or return the html.
+ *
+ * @return string $html Compiled html based on our arguments.
```
Comment (ntwb) [inline inc/template-tags.php]:
```suggestion
 * @return string $html Compiled HTML based on our arguments.
```

Comment (ntwb) [reply inc/template-tags.php]: Uppercase HTML for consistency with the other instances of this in this PR

Comment (nielslange) [reply inc/template-tags.php]: @itowhid06 In line 28 there's another `html` which with should be `HTML`.

**Pull request merged**